bitswap/src/ledger.rs

```diff
@@ -108,6 +108,7 @@ impl Ledger {
         if self.message.is_empty() {
             return None;
         }
+        // FIXME: this might produce too large message
```

Comment:
```suggestion
        // FIXME: this might produce too large messages
```

Reply:
I think we are only producing a single one here?

Reply:
ah, then "too large a message" would be the one

Reply:
Then you want "too large of a message" or maybe "this might produce an oversized message"

Reply:
Fight me :laughing: https://en.wikipedia.org/wiki/English_articles#Word_order:
> Adjectives qualified by too, so, as and how generally precede the indefinite article: too great a loss, so hard a problem, as delicious an apple as I have ever tasted, I know how pretty a girl she is.

Reply:
Follow any of these so-called rules? Native English speakers like me never, as proper and formal as they may be!

Reply:
But yeah you can drop the "of", I think we had a little comment race there too

Reply:
> Follow any of these so-called rules?

I think I just did :smile:; I don't even know where I got this from, I had to find the rule to substantiate my claim :laughing: